src/test/java/org/fundacionjala/coding/melvi/EvaporatorTest.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.melvi;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class EvaporatorTest {
+
+    public static final int EXPECTED = 22;
+    public static final int INT = 10;
+    public static final int THRESHOLD = 20;
+    public static final int EXPECTED1 = 16;
```

Comment:
These constants should be defined as local final variables inside test method. (expectedResult)